docs/api/IoT.js-API-WebSocket.md

```diff
@@ -0,0 +1,92 @@
+### Platform Support
+
+The following chart shows the availability of each WebSocket module API function on each platform.
+
+|  | Linux<br/>(Ubuntu) | Tizen<br/>(Raspberry Pi) | Raspbian<br/>(Raspberry Pi) | Nuttx<br/>(STM32F4-Discovery) | TizenRT<br/>(Artik053) |
+| :---: | :---: | :---: | :---: | :---: | :---: |
+| websocket.connect | O | O | O | X | O |
+| websocket.close | O | O | O | X | O |
+| websocket.ping | O | O | O | X | O |
+| websocket.send  | O | O | O | X | O |
+
+# WebSocket
+
+WebSocket provides full-duplex communication over a TCP connection. It is designed to work over HTTP ports 80 and 443.
+
+### Requirements
+WebSocket requires you to enable both the `TLS` and the `WebSocket` module. This can be done by compiling IoT.js with `--cmake-param=-DENABLE_MODULE_WEBSOCKET=ON`. Currently WebSocket only works if TLS is enabled as well.
```

Comment:
Is TLS required? Becaose of crypto? Meybe we should relax this requirement in the future.

Reply:
Yes. In the future I would like to port mbedTLS's functions to be externally available by a crypto API.